docs/openstack-servers-images-default-users.md

```diff
@@ -0,0 +1,23 @@
+# Default users for each Distro
+
+| Image Name              | Initial User |
+|-------------------------|--------------|
+| AlmaLinux-8.9           | almalinux    |
+| AlmaLinux-9.4           | almalinux    |
+| ArchLinux-2024.05.15    | arch         |
+| CentOS-Stream-8         | cloud-user   |
+| CentOS-Stream-9         | cloud-user   |
+| Debian-11               | debian       |
+| Debian-12               | debian       |
+| Fedora-40               | fedora       |
+| Fedora-CoreOS-40        | core         |
+| FreeBSD-14.0-ufs        | freebsd      |
+| FreeBSD-14.0-zfs        | freebsd      |
+| magnum-fedora-coreos-40 | coreos       |
+| NixOS-24.05             | nixos        |
+| Rocky-8.9               | rocky        |
+| Rocky-9.4               | rocky        |
+| Ubuntu-20.04            | ubuntu       |
+| Ubuntu-22.04            | ubuntu       |
+| Ubuntu-24.04            | ubuntu       |
+| Test                    | test         |
```

Comment:
Do we need to have the Test image?

Reply:
Sorry, left over from initial table formatting.